servers/sklearnserver/sklearnserver/requirements.txt

```diff
@@ -1,4 +1,4 @@
 seldon_core
-scikit-learn == 0.24.2
-numpy >= 1.8.2
+scikit-learn == 1.1.0
```

Comment:
Would it be possible to keep the previous versions?

Reply:
Yes it will be possible.

Reply:
I would keep it to the same versions as in `master` to reduce the number of changes

Reply:
i.e.

`scikit-learn==0.24.2`, `numpy>=1.8.2` and same would apply to the xgboost reqs

Reply:
@adriangonz sure let me revert back changes.

Reply:
@adriangonz in master for xgboost server the requirements are follows should I keep it same ?
```
scikit-learn == 0.20.3
numpy >= 1.8.2
xgboost == 1.4.2
```

Reply:
Yeah, let's keep the same as in `master` for both

Reply:
@adriangonz Done.